webapp/src/js/app.js

```diff
@@ -107,6 +107,36 @@ _.templateSettings = {
     return;
   }
 
+  var ONLINE_ROLE = 'mm-online';
+  var ONLINE_USER_DENIED_ROUTES = ['/tasks', '/messages', '/contacts'];
```

Comment:
This should be expanded for at least each of the main tabs, so add "analytics" and "contacts" as well as anything else you can think of.